pubsub/api/Pubsub.Samples.Tests/PubsubFixture.cs

```diff
@@ -24,7 +24,10 @@ public class PubsubFixture : IDisposable, ICollectionFixture<PubsubFixture>
     public string ProjectId { get; }
     public List<string> TempTopicIds { get; } = new List<string>();
     public List<string> TempSubscriptionIds { get; } = new List<string>();
+    public List<string> TempSchemaIds { get; } = new List<string>();
```

Comment:
I think you forgot to delete these on Dispose.

Reply:
Schemas are forever! No, just kidding. Done.